### ISSUE TITLE
Spacy Package for Urdu Language Model

### ISSUE THREAD

**Issue by zarmeen92** [avineshpvs/indic_tagger#1]
Are you planning to release Spacy package for Urdu language as done for TE?

**Comment by avineshpvs** [comment]
Yes, we are planning to do it for all the languages. However, mind you that the SpaCy models do not have the accuracy of the CRF models.
